mmengine/visualization/visualizer.py

```diff
@@ -384,6 +385,7 @@ def draw_points(self,
                 marker styles. Defaults to None.
             sizes (Optional[Union[np.ndarray, torch.Tensor]]): The marker size.
                 Defaults to None.
+            zorder (float): The z-order of points. Defaults to 1.0.
```

Comment:
Refer to the description here(https://matplotlib.org/stable/gallery/misc/zorder_demo.html) and refine the docstring